### PR TITLE
method names changed in asyncAirtable.js, tests, and readme

### PR DESCRIPTION
**Pull request by mamacas**

### CODE REVIEW

**Comment by mamacas** [comment]
won't let me work on tests without an API key, sorry!

**Comment by GV14982** [comment]
Hi,

Please also change the main.test.js file so it has all the correct method names.

File: lib/asyncAirtable.js [hunk 1]
```diff
@@ -190,13 +190,13 @@ class AsyncAirtable {
...
 
-  delete = async (table, id) => {
+  deleteRecord = async (table, id) => {
     try {
       this.checkArg(table, 'table', 'string', true);
       this.checkArg(id, 'id', 'string', true);
       let url = `${baseURL}/${this.base}/${table}/${id}`;
       const res = await nodeFetch(url, {
-        method: 'delete',
+        method: 'deleteRecord',
```
Comment (GV14982) [inline lib/asyncAirtable.js:199]: Please change this back to delete, as deleteRecord is not a valid HTTP method.

File: lib/asyncAirtable.js [hunk 2]
```diff
@@ -282,7 +282,7 @@ class AsyncAirtable {
       });
       let url = `${baseURL}/${this.base}/${table}?${encodeURI(query)}`;
       const res = await nodeFetch(url, {
-        method: 'delete',
+        method: 'deleteRecord',
```
Comment (GV14982) [inline lib/asyncAirtable.js:285]: Please change this back to delete, as deleteRecord is not a valid HTTP method.

**Review by GV14982 — CHANGES_REQUESTED** [review]
Please change the HTTP methods on deleteRecord and bulkDelete back to delete, as deleteRecord is not a valid HTTP method.

Thanks!

**Comment by GV14982** [comment]
Handled with #15

**Pull request closed without merging**